src/main/java/org/jenkinsci/test/acceptance/po/Node.java

```diff
@@ -44,6 +44,8 @@ protected Node(Injector i, URL url) {
     public abstract String getName();
 
     public void setExecutors(int n) {
+        // On high Res screens the UI redraws, so best wait for numExecutors to be present
+        waitFor(by.path("/numExecutors"));
```

Comment:
But this can affect any control on any config form, no? Also, are we sure redrawing is the cause - does it remove/hide the element temporarily?